bindings/python/CMakeLists.txt

```diff
@@ -31,13 +31,15 @@ set_property(
     TARGET ${target_name}
     PROPERTY SWIG_DEPENDS python.i numpy.i)
 
-set_property(
-    TARGET ${target_name}
-    PROPERTY SWIG_COMPILE_OPTIONS -Wextra -threads)
-
-set_property(
-    TARGET ${target_name}
-    PROPERTY SWIG_GENERATED_COMPILE_OPTIONS -Wextra)
+if(NOT MSVC)
```

Comment:
Sorry if it's a dumb question but why do we use here the flag `MSVC` (Visual Studio compiler) instead of the `WIN32` (more generic?) ?

Reply:
The question is not dump at all! On Windows, it is possible to use gcc (typically via the mingw-w64 distribution) or clang, and in those cases the use of GCC/clang options such as `-Wextra` make sense. We just want to avoid to pass this options if MSVC is used, not if gcc or clang are used on Windows, hence the use of `MSVC` instead of `WIN32`.